R/simulation-run-options.R

```diff
@@ -21,6 +21,16 @@ SimulationRunOptions <- R6::R6Class(
       self$numberOfCoresToUse <- numberOfCoresToUse
       self$checkForNegativeValues <- checkForNegativeValues
       self$showProgress <- showProgress
+    },
+    #' @description
+    #' Print the object to the console
+    #' @param ... Rest arguments.
+    print = function(...) {

```

Comment:
@copy from your code

Reply:
I don't think I wrote this.

Reply:
Sorry this was meant to be @PavelBal  copy. Cheers